### ISSUE TITLE
Letter 'f', the horizontal bar is placed too high  

### ISSUE THREAD

**Issue by sewkokot** [belluzj/fantasque-sans#126]
The more I use fantasque-sans, the more I like it :)
I prefer non-looped k, so thank you for this option.
One think that bothers me now is the shape of letter f. For me the horizontal bar is placed too high, and using small font size, the top curved shape and the bar are a bit too lumped. Therefore I would be interested in lowering the bar and having it as an option or stylistic set?

**Comment by RedBearAK** [comment]
I agree with this. At smaller sizes the horizontal bar and the top of the "f" are a bit too close together.
